### PR TITLE
Simplifies some parameters and functions.

### PR DESCRIPTION
**Pull request by armfazh**
Solving some issues that I detected during the review.

### CODE REVIEW

File: draft-irtf-cfrg-voprf.md [hunk 1]
```diff
@@ -602,7 +600,7 @@ required.
 ~~~
 struct {
   SerializedElement elements<1..2^16-1>;
-  Scalar proof<0...2^16-1>; /* optional */
+  Scalar proof[2]; /* only for modeVerifiable */
```
Comment (chris-wood) [inline draft-irtf-cfrg-voprf.md:603]: Please revert this -- the old notation was correct!

Comment (armfazh) [reply draft-irtf-cfrg-voprf.md:603]: What this notation `proof<0...2^16-1>` means?

Comment (chris-wood) [reply draft-irtf-cfrg-voprf.md:603]: It's TLS notation -- it means that `proof` is a list of `Scalar` elements consisting of anywhere from 0 to 2^16-1 bytes.

File: draft-irtf-cfrg-voprf.md [hunk 2]
```diff
@@ -602,7 +600,7 @@ required.
 ~~~
 struct {
   SerializedElement elements<1..2^16-1>;
-  Scalar proof<0...2^16-1>; /* optional */
+  Scalar proof<0...2^16-1>; /* only for modeVerifiable */
```
Comment (alxdavids) [inline draft-irtf-cfrg-voprf.md:603]:
```suggestion
  Scalar proof<0...2^16-1>; /* only in verifiable context */
```

**Pull request merged**